pylint/checkers/base_checker.py

```diff
@@ -193,21 +184,10 @@ def check_consistency(self) -> None:
     def create_message_definition_from_tuple(
         self, msgid: str, msg_tuple: MessageDefinitionTuple
     ) -> MessageDefinition:
-        with warnings.catch_warnings():
-            warnings.filterwarnings("ignore", category=DeprecationWarning)
-            if isinstance(self, (BaseTokenChecker, BaseRawFileChecker)):
-                default_scope = WarningScope.LINE
-            # TODO: 3.0: Remove deprecated if-statement
-            elif implements(self, (IRawChecker, ITokenChecker)):
-                warnings.warn(  # pragma: no cover
-                    "Checkers should subclass BaseTokenChecker or BaseRawFileChecker "
-                    "instead of using the __implements__ mechanism. Use of __implements__ "
-                    "will no longer be supported in pylint 3.0",
-                    DeprecationWarning,
-                )
-                default_scope = WarningScope.LINE  # pragma: no cover
-            else:
-                default_scope = WarningScope.NODE
+        if isinstance(self, (BaseTokenChecker, BaseRawFileChecker)):
```

Comment:
```suggestion
        if isinstance(self, (BaseTokenChecker, BaseRawFileChecker, IRawChecker, ITokenChecker):
```
Should we do this here @DanielNoord ?

Reply:
No I was mistaken. Those should be removed. Everything that gets tested for Deprecation should be removed indeed!